content/docs/guides/error.md

```diff
@@ -18,6 +18,52 @@ If an error occurs, gRPC returns one of its error status codes instead, with an
 optional string error message that provides further details about what happened.
 Error information is available to gRPC clients in all supported languages.
 
+### Advanced error modeling
+
+The error model described above is the official gRPC error model,
+is supported by all gRPC client/server libraries, and is independent of
+the gRPC data format (whether protocol buffers or something else). You
+may have noticed that it's quite limited and doesn't include the
+ability to communicate error details.
+
+If you're using protocol buffers as your data format, however, many of
+the [gRPC language libraries](https://grpc.io/docs/reference/) now
+provide experimental support for the richer error model developed and
+used by Google as described
+[here](https://cloud.google.com/apis/design/errors#error_model). This
+model enables servers to return and clients to consume additional
+error details expressed as one or more protobuf messages. It further
+specifies a [standard set of error message
+types](https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto)
+to cover the most common needs (such as invalid parameters, quota
+violations, and stack traces). The protobuf binary encoding of this
+extra error information is provided as trailing metadata in the
+response.
+
+**TODO: this [SO post](https://stackoverflow.com/a/48750825) states
+that "Support is coming to each language to handle the type". What can
+be said here about that commitment and status/timeline?**
+
```

Comment:
@ctoomey, since there is no timeline to add this in all languages (regardless of what the comment says in the post), it won't be easy to get an answer to this question. Please either remove this TODO or replace it with something like "Currently, there is no plan to natively support this in all languages as some gRPC implementations do not take a dependency on protocol buffers. Some relevant information can be found [here](https://github.com/grpc/proposal/blob/master/L44-python-rich-status.md) and  [here](https://jbrandhorst.com/post/grpc-errors/)."

Reply:
@srini100 from digging around in the various github repos it seems this is already supported in C++, Go, Java, Python, and Ruby, and there are open issues requesting and discussing it for at least Node and grpc-web. I'm guessing it's only the grpc-core library in C that doesn't depending on protobuf, so support might still be added for the remaining languages?

It'd be a lot more helpful to developers to provide this additional detail. How 'bout this?

> This richer error model is already supported in the C++, Go, Java, Python, and Ruby libraries, and at least the grpc-web and Node.js libraries have open issues requesting it. Other language libraries may add support in the future if there's demand, so check their github repos if interested. Note however that the grpc-core library written in C will not likely ever support it since it is purposely data format agnostic.

Reply:
This looks fine.